confection/tests/test_config.py

```diff
@@ -1396,3 +1396,11 @@ def test_warn_single_quotes():
     commands = some'thing
     """
     cfg = Config().from_str(str_cfg)
+
+
+@pytest.mark.skip(reason="Behavior will be fixed in future PR")
```

Comment:
I think we want this test to be marked with `xfail` instead?

Reply:
Fixed in https://github.com/explosion/confection/pull/32/commits/b931591883a211998910cd6c339b56398646129d.